packages/database/src/tests/changeHandlers/SurveyResponseOutdater.test.js

```diff
@@ -3,66 +3,504 @@
  * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
  */
 
+/* eslint-disable camelcase */
+
 import { expect } from 'chai';
-import sinon from 'sinon';
 
-import { sleep } from '@tupaia/utils';
 import { SurveyResponseOutdater } from '../../changeHandlers/SurveyResponseOutdater';
-import { getTestModels } from '../../testUtilities/getTestDatabase';
-import { buildAndInsertSurveys, buildAndInsertSurveyResponses } from '../../testUtilities';
+import {
+  buildAndInsertSurveys,
+  findOrCreateDummyRecord,
+  getTestModels,
+  generateTestId,
+  populateTestData,
+  upsertDummyRecord,
+} from '../../testUtilities';
+
+const buildSurvey = (id, periodGranularity) => {
+  const code = `Test_${periodGranularity || 'no_granularity'}`;
 
-const SURVEY = {
-  code: 'Test_Outdater',
-  questions: [{ code: 'Test_Outdater1', type: 'Number' }],
+  return {
+    id,
+    code,
+    period_granularity: periodGranularity,
+    questions: [{ code: `${code}1`, type: 'Number' }],
+  };
+};
+
+const yearlySurveyId = generateTestId();
+const quarterlySurveyId = generateTestId();
+const monthlySurveyId = generateTestId();
+const weeklySurveyId = generateTestId();
+const dailySurveyId = generateTestId();
+const nonPeriodicSurveyId = generateTestId();
+
+const userId = generateTestId();
+
+const SURVEYS = {
```

Comment:
I moved the test cases from `testOutdatedStatusUpdate.js` in this file, and moved the existing test cases (which asserted generic change handling behaviour) in `ChangeHandler.test.js`